checkout-v3/enterprise/seamless-view.md

```diff
@@ -102,7 +102,7 @@ request.addEventListener('load', function () {
         // Payment Menu inside 'checkout-container'.
         payex.hostedView.checkout({
             container: {
-                checkoutContainer: "checkout-container"
+                checkout: "checkout-container"
```

Comment:
```suggestion
                checkout: "checkout"
```
I see that `-container` is removed from the equivalent example in [checkout-v3/business/seamless-view.md]
(https://github.com/SwedbankPay/developer.swedbankpay.com/pull/1391/files#diff-3d5ee9b314e3af1a65fdde678bb35fa9976c4c8dbee065af9bd6521853ee278b). Is that the case here  as well?